client/source/App.jsx

```diff
@@ -28,16 +29,19 @@ const App = () => {
       });
   }
 
+  const [prodID, setProdID] = useState('product id');
```

Comment:
if you use the suggestion I made below regarding the `.Provider` method, you'll need to `useState` / `setState` in the `index.jsx` file as well